api/peripheral_api/routes/Printer.js

```diff
@@ -6,22 +6,30 @@ const {
   UNAUTHORIZED,
   NOT_FOUND
 } = require('../../util/constants').STATUS_CODES;
-const s3BucketKeys = require('../../config/config.json').S3Bucket;
-const printingS3Bucket = require('../../config/config.json').PrintingS3Bucket;
+const { s3BucketKeys, printingS3Bucket, DISCORD_PRINTING_KEY }
+= require('../../config/config.json');
 const {
+  CLIENT_ID,
+  CLIENT_SECRET,
   ACCOUNT_ID,
-  PAPER_PRINTING_QUEUE_NAME
+  PAPER_PRINTING_QUEUE_NAME,
 } = require('../../config/config.json').Queue;
 const {
   verifyToken,
-  checkIfTokenSent
+  checkIfTokenSent,
+  checkDiscordKey
 } = require('../../util/token-verification');
+const{ SceSqsApiHandler } = require('../util/SceSqsApiHandler');
+
+const SqsHandler = new SceSqsApiHandler(PAPER_PRINTING_QUEUE_NAME);
```

Comment:
i would rather have this be initialized in the API handler instead of being global, up to you, also use camelcase (`sqsHandler`)